context_test.go

```diff
@@ -341,49 +334,49 @@ func TestDetectProjectGOPATH(t *testing.T) {
 		{
 			name:         "project-with-no-AbsRoot",
 			root:         "",
-			resolvedRoot: filepath.Join(ctx.GOPATHs[0], "src", "real", "path"),
+			resolvedRoot: filepath.Join(ctx.GOPATHs[0], "src/real/path"),
```

Comment:
cool, ty; yes, it does call `filepath.Clean()`, but that func doesn't do separator transformations.